src/platform-includes/sourcemaps/primer/javascript.nextjs.mdx

```diff
@@ -0,0 +1,5 @@
+To show readable stack traces in your Sentry errors, once configured, the `@sentry/nextjs` SDK will generate and upload source maps automatically.
+
+To upload source maps, the `@sentry/nextjs` SDK uses the [Sentry Webpack Plugin](https://github.com/getsentry/sentry-webpack-plugin) under the hood. See the [Manual Configuration](../manual-setup/) page and the Sentry [Webpack Plugin documentation](https://github.com/getsentry/sentry-webpack-plugin#cli-configuration) for more details.
```

Comment:
Should we link them specifically to the sourcemaps section of the manual setup page?